wg-api/best-practices.md

```diff
@@ -32,6 +32,15 @@ function whatever(opts: { a: string, b?: boolean }) { /* ... */ }
 
 See https://w3ctag.github.io/design-principles/#prefer-dict-to-bool for more details.
 
+### How will third-party libraries interact with this API?
```

Comment:
I wonder if the guidelines are applied to only 3rd party libraries? Perhaps we can make the title more generic like "How will different components interact with this API?" because I feel like this is helpful for the API author to think about not just 3rd party libraries, but also multiple internal modules interacting, or plugins or extensions.